### DISCUSSION TITLE
France 3 Bretagne [FR]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#900] [Q&A]
### Channel ID

France3Bretagne.fr

### Notes (optional)

It airs shows in breton language

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
online not geoblocked
http://www.callofliberty.fr/tv/207-france.3bretagne/207-france.3bretagne.php


#EXTINF:-1 ,france.3 bretagne geoblocked
#EXTVLCOPT:http-referrer=http://www.callofliberty.fr/tv/207-france.3bretagne/207-france.3bretagne.php
http://www.callofliberty.fr/stream/france.3bretagne/master.m3u8

Kodi
#EXTINF:-1 ,france.3 bretagne geoblocked
http://www.callofliberty.fr/stream/france.3bretagne/master.m3u8|Referer=http://www.callofliberty.fr/tv/207-france.3bretagne/207-france.3bretagne.php

same trick can be done whit all other local FR3 channel  from the site

#EXTINF:0,210. France 3 Alpes
http://livetv.ktv.zone/210/play.m3u8
#EXTINF:0,211. France 3 Alsace
http://livetv.ktv.zone/211/play.m3u8
#EXTINF:0,212. France 3 Aquitaine
http://livetv.ktv.zone/212/play.m3u8
#EXTINF:0,213. France 3 Auvergne
http://livetv.ktv.zone/213/play.m3u8
#EXTINF:0,216. France 3 Bretagne
http://livetv.ktv.zone/216/play.m3u8
#EXTINF:0,218. France 3 Champagne-Ardenne
http://livetv.ktv.zone/218/play.m3u8
#EXTINF:0,220. France 3 Côte Azur
http://livetv.ktv.zone/220/play.m3u8
#EXTINF:0,223. France 3 Languedoc
http://livetv.ktv.zone/223/play.m3u8
#EXTINF:0,224. France 3 Limousin
http://livetv.ktv.zone/224/play.m3u8
#EXTINF:0,225. France 3 Lorraine
http://livetv.ktv.zone/225/play.m3u8
#EXTINF:0,226. France 3 Midi-Pyrénées
http://livetv.ktv.zone/226/play.m3u8
#EXTINF:0,227. France 3 Nord Pas-de-Calais
http://livetv.ktv.zone/227/play.m3u8
#EXTINF:0,230. France 3 Picardie
http://livetv.ktv.zone/230/play.m3u8
#EXTINF:0,231. France 3 Poitou-Charentes
http://livetv.ktv.zone/231/play.m3u8
#EXTINF:0,232. France 3 Provence-Alpes
http://livetv.ktv.zone/232/play.m3u8
#EXTINF:0,233. France 3 Rhône-Alpes
http://livetv.ktv.zone/233/play.m3u8

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19867
